### DISCUSSION TITLE
Server-side/client-side for plugins

### DISCUSSION THREAD

**Discussion by Madis0** [modrinth/.github#89] [Feature requests]
**Is your feature request related to a problem? Please describe.**
I know there aren't any clientside plugins, but there are plugins that depend on clientside mods. As a server admin I would like to either avoid or only see those in the search, so they need to have that marked on the listing.

**Describe the solution you'd like**
A marking similar to mods - server-side only or required on server and client.

**Describe alternatives you've considered**
A tag that describes it? Though it's not possible to _exclude_ any tags from search filters yet, so it doesn't really solve the issue for me.

**Additional context**

**Comment by Andre601** [comment]
I would say the majority of these plugins usually display a text like "Universal mod and plugin" under their name.

And if not then they most likely mention a dependency somewhere within their description.

**Comment by Madis0** [comment]
> I would say the majority of these plugins usually display a text like "Universal mod and plugin" under their name.
> 
> And if not then they most likely mention a dependency somewhere within their description.

That is all good for not getting misled, but it doesn't help with search filtering.

**Comment by ghost** [comment]
For example, if a plugin integrates with a client mod, it could mark it as "optional" in version dependencies, but if the plugin itself can't be installed on the client for extra functionality (which it never can be) it will be client-unsupported. There is a case specific thingy for plugins that have their official mod versions.
